### PR TITLE
Add script to connect to current prod/staging container 

### PR DESCRIPTION
**Pull request by FinnIckler**
I used the same script as in wca live, but added an environment parameter and some checks if the prerequisites are installed.

### CODE REVIEW

**Comment by FinnIckler** [comment]
@jonatanklosko as you did the original script. Any comments/reviews?

File: rel/console.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+usage() {
+  printf "${COLOR_DEFAULT}Usage: $0 -e <environment>\n"
+  printf " -e <environment>  Specify the environment (production or staging)"
+  exit 1
+}
+
+# Check if AWS CLI is installed
+printf "${COLOR_DEFAULT}AWS CLI | "
+command -v aws >/dev/null 2>&1 && status="$?" || status="$?"
+if [[ "${status}" = 0 ]]; then
```
Comment (jonatanklosko) [inline rel/console.sh:26]: I believe you can do this:

```suggestion
if command -v aws >/dev/null 2>&1; then
```

File: rel/console.sh [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  printf "${COLOR_GREEN}OK ${COLOR_DEFAULT}(${smpVersion})\n"
+else
+  printf "${COLOR_RED}Missing\n"
+  exit 1;
+fi
+
+# Check if you are connected to the AWS WCA Account
+printf "${COLOR_DEFAULT}aws wca credentials | "
+account=$(aws iam list-account-aliases --query "AccountAliases[0]" --output text )
+if [[ "${account}" = "thewca" ]]; then
```
Comment (jonatanklosko) [inline rel/console.sh:48]:
```suggestion
if [[ "${account}" == "thewca" ]]; then
```

With `[[` you can use `==`, which is probably more intuitive. Alternatively, you can do `[ "${account}" = "thewca" ]`, which is sh-compatible. Either way is fine though!

File: rel/console.sh [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  usage
+fi
+
+# Set container name based on environment
+service_name="wca-on-rails-staging"
+container_name="rails-staging"
+if [ "$environment" = "production" ]; then
+  service_name="wca-on-rails-prod"
+  container_name="rails-production"
+fi
```
Comment (jonatanklosko) [inline rel/console.sh:91]: You can combine these:

```suggestion
case "$environment" in
  "production")
    service_name="wca-on-rails-prod"
    container_name="rails-production"
  ;;
  
  "staging")
    service_name="wca-on-rails-prod"
    container_name="rails-production"
  ;;

  *)
    printf "${COLOR_RED}Invalid environment: $environment. Must be 'production' or 'staging'.\n"
    usage
  ;;
esac
```

**Review by jonatanklosko — APPROVED** [review]
Just a few code suggestions, but LGTM!

File: rel/console.sh
Comment (jonatanklosko) [inline rel/console.sh:1]: Oh sorry, bad copy paste, good you caught it!

File: rel/console.sh (hunk 3 above)
Comment (jonatanklosko) [reply rel/console.sh]: Oh sorry, bad copy paste, good you caught it!

File: rel/console.sh [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  printf "${COLOR_GREEN}OK ${COLOR_DEFAULT}(${cliVersion})\n"
+else
+  printf "${COLOR_RED}Missing\n"
+  exit 1
+fi
+
+# Check whether the Session Manager plugin exists
+printf "${COLOR_DEFAULT}Session Manager Plugin | "
+command -v session-manager-plugin >/dev/null 2>&1 && status="$?" || status="$?"
+if [[ "${status}" = 0 ]]; then
```
Comment (jonatanklosko) [inline rel/console.sh:36]: Same as above :D

```suggestion
if command -v session-manager-plugin >/dev/null 2>&1; then
```

**Review by jonatanklosko — APPROVED** [review]

**Pull request merged**